lighthouse-core/audits/seo/canonical.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
@paulirish you always check those 😉 so here is a fun riddle - I created this file in 2017, but it will be merged into master in 2018. Should I put 2017 or 2018 here?

Reply:
2017 sgtm. :)